samples/bot-message-reminder/nodejs/Readme.md

```diff
@@ -0,0 +1,82 @@
+## Message reminder with messaging extension action
+
+This sample shows feature where user can schedule a task from messaging extension action and get a reminder card at scheduled time.
+
+![Select message](Images/MessageAction.png)
```

Comment:
Can you please crop this image and show only relevant section:
![image](https://user-images.githubusercontent.com/59052547/141988044-f48d2b4e-b34a-415d-a80f-23d7d1aeb070.png)